### ISSUE TITLE
Replace netplugin_port with listen_url's port

### ISSUE THREAD

**Issue by dvavili** [contiv/ansible#352]
All the playbooks are opening up netplugin_port, but the actual port that netmaster is using is the listen_url port and the control_url port.
- Replace/remove netplugin_port with listen_url and control_url port or make sure that they are the same
- Open up these ports as necessary

**Comment by rhim** [comment]
@DivyaVavili does this result in a bug if different listen/control ports are used? What's the value of netplugin_port?

**Comment by dvavili** [comment]
Yes, this will be a bug if they are different. Currently, all these vars default to 9999 port. But we should be opening up listen_url/control_url.
